src/hooks/use-sync-states-progress-info.ts

```diff
@@ -17,6 +18,12 @@ export type PullStateProgressInfoValues = Record<
 	PullStateProgressInfo
 >;
 
+export type SyncBackupResponse = {
+	status: 'in-progress' | 'finished' | 'failed';
+	download_url: string;
+	percent: number;
+};
+
```

Comment:
This type is also defined in `src/hooks/sync-sites/use-sync-pull.ts`. Could we use a single definition, and, while we're at it, maybe convert it to a zod schema?

Reply:
For reference, I've deleted the redundant definition in this commit [2241fd6](https://github.com/Automattic/studio/pull/1380/commits/2241fd6791019219722eb00aa79c85f9e16c638f)